modules/lvms-creating-volume-snapshots.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-snapshots_{context}"]
+= Creating volume snapshots
+
+You can create volume snapshots based on the available capacity of the thin pool and the over-provisioning limits.
+To create a volume snapshot, you must create a `VolumeSnapshotClass` custom resource (CR).
+
+.Prerequisites
+
+* You ensured that the persistent volume vlaim (PVC) is in `Bound` state. This is required for a consistent snapshot.
+* You stopped all the I/O to the PVC.
+
+.Procedure
+
+. Log in to the OpenShift CLI (`oc`).
++
+[IMPORTANT]
+====
+If you are using {lvms} in {rh-rhacm}, log in to the {rh-rhacm} CLI using your {product-title} credentials.
+====
+
+. Create a `VolumeSnapshotClass` CR YAML file:
++
+.Example `VolumeSnapshotClass` CR to create a volume snapshot
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+    name: lvm-block-1-snap <1>
+spec:
+    volumeSnapshotClassName: lvms-vg1 <2>
+    source:
+        persistentVolumeClaimName: lvm-block-1
+----
+<1> Specify a name for the volume snapshot.
+<2> Set this field to the value of the `storageClassName` field in the `PersistentVolumeClaim` CR of the source PVC.  
```

Comment:
```suggestion
<2> Set this field to the name of either default `volumeSnapshotClassName` present in cluster or custom `volumeSnapshotClassName` 
```

Reply:
@radeore 
Is this field ``volumeSnapshotClassName` nothing but the name of the volume snapshot?

Reply:
Updated as per the suggestion.